presto-parquet/src/main/java/com/facebook/presto/parquet/reader/PageReader.java

```diff
@@ -60,12 +77,15 @@ public DataPage readPage()
         }
         DataPage compressedPage = compressedPages.removeFirst();
         try {
+            long firstRowIndex = getFirstRowIndex(pageIndex++, offsetIndex);
```

Comment:
how about not put `++` in parameter passing? put it outside

Reply:
If it is the convention of Presto community, sure.